recipes-debian/net-tools/net-tools_debian.bb

```diff
@@ -0,0 +1,88 @@
+# base recipe: poky/meta/recipes-extended/net-tools/net-tools_1.60-26.bb
+# base branch: warrior
+# base commit: fec5323dba7e23a42073995020c7336f3e6a7de1
+
+SUMMARY = "Basic networking tools"
+DESCRIPTION = "A collection of programs that form the base set of the NET-3 networking distribution for the Linux operating system"
+HOMEPAGE = "http://net-tools.berlios.de/"
+BUGTRACKER = "http://bugs.debian.org/net-tools"
+LICENSE = "GPLv2+"
+LIC_FILES_CHKSUM = "file://COPYING;md5=b234ee4d69f5fce4486a80fdaf4a4263 \
+                    file://ifconfig.c;beginline=11;endline=15;md5=d1ca372080ad5401e23ca0afc35cf9ba"
+
+inherit debian-package
+require recipes-debian/sources/net-tools.inc
+
+FILESPATH_append = ":${COREBASE}/meta/recipes-extended/net-tools/net-tools"
+SRC_URI += "file://net-tools-config.h \
+            file://net-tools-config.make \
+            file://musl-fixes.patch \
+            file://net-tools-1.60-sctp1.patch \
+           "
+
+inherit gettext
+
+do_patch[depends] += "quilt-native:do_populate_sysroot"
+
+# The Makefile is lame, no parallel build
+PARALLEL_MAKE = ""
+
+do_unpack[cleandirs] += "${S}"
+
+# We invoke base do_patch at end, to incorporate any local patch
+python do_patch() {
```

Comment:
@AkitoTakada Can this function remove ? poky's recipe calls nettools_do_patch in do_patch task, so it needs to call patch_do_patch. However, this recipe doesn't call nettools_do_patch, so it looks as if it's okay to remove this function.